python/rmm/docs/guide.md

```diff
@@ -236,17 +236,17 @@ Common to both usages is that they modify the currently active RMM memory resour
 
 >>> # We start with the default cuda memory resource
 >>> rmm.mr.get_current_device_resource()
-<rmm._lib.memory_resource.CudaMemoryResource at 0x7f7e6c0a1ce0>
+<rmm.pylibrmm.memory_resource.CudaMemoryResource object at 0x7fa0da48a8e0>
```

Comment:
These changes were a good find. How did you notice them? Have you grepped the code base for other instances of `rmm._lib`? That would probably be good to do if you haven't.

Reply:
Yes, I grepped for `rmm._lib` and replaced. But I missed the `.gitignore` file, I'll replace there too